Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/export-settings.js

```diff
@@ -5,6 +5,7 @@ angular.module('virtoCommerce.exportModule')
         blade.isLoading = true;
         blade.exportDataRequest = blade.exportDataRequest || {};
         blade.isExportedTypeSelected = typeof (blade.exportDataRequest.exportTypeName) !== 'undefined';
+        blade.knowntypes = [];
```

Comment:
Probably here we do not need to fill it with empty array, as it is valid value even after getting types. Let's fill it with null or leave undefined.